### PR TITLE
No response bot

### LINKED ISSUE #467: Set up a bot which closes support issues automatically
🐞 **Problem**

The issues labeled with `Support` has to be closed when the author/us stop interacting on the issue for 3 weeks.

🎯 **Goal**

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request.
 * Add this in the comments:
 	- `Addresses #<put issue number here>` if you are partially fixing the issue.
 	- `Fixes #<put issue number here>` if you are completely fixing the issue.
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by yauneyz**
Fixes #467

Fixed the typo - how embarrassing

@shreyasbapat

### CODE REVIEW

**Comment by shreyasbapat** [comment]
I will merge this tomorrow

**Comment by yauneyz** [comment]
Terrific

File: .github/no-response.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#Configuration for probot-no-response - https://github.com/probot/no-response
+
+#Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 21
```
Comment (ritzvik) [inline .github/no-response.yml]: @shreyasbapat Very auspicious number it is !

Comment (shreyasbapat) [reply .github/no-response.yml]: I know right?

**Pull request merged**